pybsn/__init__.py

```diff
@@ -423,7 +565,10 @@ def _attempt_modern_login(session, url, username, password):
         response.raise_for_status()
 
 
-def connect(host, username=None, password=None, token=None, login=None, verify_tls=False, session_headers=None):
+def connect(host, username=None, password=None, token=None, login=None,
+            verify_tls=False, session_headers=None,
+            timeout=None,
+            *args, **kwargs) -> BigDbClient:
```

Comment:
`*args`, `**kwargs` - will cause connect to accept arbitrary arguments with no warnings/IDE indication. Why would we want that?

Reply:
cleaned up in the ce4bbebb4a37bfdf69da6c28306350f76a21588e

Reply:
Thanks!